.github/workflows/build.yml

```diff
@@ -43,31 +43,17 @@ jobs:
           gclient config --name=src --unmanaged https://github.com/${{ github.repository }}
           gclient sync -v --no-history --shallow
 
-      - name: Generate Tizen 6.0 sysroot
-        if: ${{ matrix.api-version == '6.0' }}
-        run: src/tools/generate_sysroot.py --api-version 6.0 --out src/sysroot-6.0
-
       - name: Generate Tizen 6.5 sysroot
         if: ${{ matrix.api-version == '6.5' }}
         run: src/tools/generate_sysroot.py --api-version 6.5 --out src/sysroot-6.5
 
-      - name: Build for Tizen 5.5
-        if: ${{ matrix.api-version == '5.5' }}
-        run: |
-          src/tools/gn \
-            --target-cpu ${{ matrix.arch }} \
-            --target-toolchain /usr/lib/llvm-12 \
-            --target-dir build
-          ninja -C src/out/build
-
       - name: Build for Tizen 6.0
         if: ${{ matrix.api-version == '6.0' }}
         run: |
           src/tools/gn \
             --target-cpu ${{ matrix.arch }} \
             --target-toolchain /usr/lib/llvm-12 \
-            --target-sysroot src/sysroot-6.0/${{ matrix.arch }} \
-            --api-version 6.0 --system-cxx \
+            --system-cxx \
```

Comment:
We could make `--system-cxx` default.

Please update `BUILDCONFIG.gn` as well as `tools/gn`. You can also remove the note in README.

Reply:
Right. I think the option name should be `--no-system-cxx`.